controllers/nstemplateset/nstemplatetier.go

```diff
@@ -77,16 +77,13 @@ type tierTemplate struct {
 
 // process processes the template inside of the tierTemplate object and replaces the USERNAME variable with the given username.
```

Comment:
We need to update the method description too.

Reply:
done in 161ef06